extension/src/background/helpers/dataStorage.ts

```diff
@@ -126,3 +127,14 @@ export const migrateSorobanRpcUrlNetworkDetails = async () => {
 
   await localStore.setItem(NETWORKS_LIST_ID, migratedNetworkList);
 }
+
+// This migration migrates the storage for custom tokens IDs to be keyed by network
+export const migrateTokenIdList = async () => {
+  const localStore = dataStorageAccess(browserLocalStorage);
+
+  const tokenIdList = await localStore.getItem(TOKEN_ID_LIST)
+
+  // intially this was assumed to be Futurenet
+  const compositeKey = `${TOKEN_ID_LIST}__${Networks.FUTURENET}`
```

Comment:
this might be a little easier to deal with if you structured it as:

```
{
   FUTURENET: {
      [KEYID1]: ["C..."],
      [KEYID2]: ["C..."]
   },
   TESTNET: {
      [KEYID1]: ["C..."]
   }
}
```

I don't think there's anything necessarily wrong with the way you're doing it, but since you can use any kind of data type here, might be more ergonomic to use one where you don't need to encode/decode this compositeKey each time you want to use it

Reply:
works for me, I refactored to this in ae537314293f7e7b93d4c612ba244371bf8f9857